File: rb/utility/make_context.rb
# TaiwanLegalAi SDK utility: make_context
require_relative '../core/context'
module TaiwanLegalAiUtilities
  MakeContext = ->(ctxmap, basectx) {
    TaiwanLegalAiContext.new(ctxmap, basectx)
  }
end

File: php/utility/MakeContext.php
<?php
declare(strict_types=1);

// TaiwanLegalAi SDK utility: make_context

require_once __DIR__ . '/../core/Context.php';

class TaiwanLegalAiMakeContext
{
    public static function call(array $ctxmap, ?TaiwanLegalAiContext $basectx): TaiwanLegalAiContext
    {
        return new TaiwanLegalAiContext($ctxmap, $basectx);
    }
}
